app/src/main/java/io/gnosis/safe/ui/settings/app/AdvancedAppSettingsFragment.kt

```diff
@@ -70,6 +74,24 @@ class AdvancedAppSettingsFragment : BaseViewBindingFragment<FragmentSettingsAppA
                     settingsHandler.allowScreenShots(window, screenshotPermission.settingSwitch.isChecked)
                 }
             }
+            trackingHelpText.appendLink(
+                url = getString(R.string.link_tracking_link),
+                urlText = getString(R.string.tracking_text),
+                underline = true,
+                linkIcon = R.drawable.ic_external_link_green_16dp
+            )
+            trackingPermission.settingSwitch.isChecked = settingsHandler.trackingAllowed
+            trackingPermission.settingSwitch.setOnClickListener {
+                settingsHandler.trackingAllowed = trackingPermission.settingSwitch.isChecked
+                settingsHandler.allowTracking(requireContext(), trackingPermission.settingSwitch.isChecked)
+            }
+
+            if (CLIENT_GATEWAY_URL.contains("staging")) {
+                debugContainer.visible(true)
+                crashTheApp.setOnClickListener {
+                    throw  RuntimeException("Deliberate Crash")
```

Comment:
too much space :)

Reply:
I don't understand. My auto-formatter does it like this.

Reply:
there are two spaces btw throw and RuntimeException

Reply:
Fixed. 
Interestingly AS' formatting doesn't changes this.